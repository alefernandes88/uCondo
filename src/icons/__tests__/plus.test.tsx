import React from 'react'
import { render } from '@testing-library/react-native'
import PlusIcon from '../plus'

describe('Icons -> Plus', () => {
  it('should render with a normal style', () => {
    const { toJSON } = render((
      <PlusIcon />
    ))

    expect(toJSON()).toMatchSnapshot()
  })

  it('should render with a custom style', () => {
    const { toJSON } = render((
      <PlusIcon
        color="red"
        height={30}
        width={30}
        strokeSize={1.5}
      />
    ))

    expect(toJSON()).toMatchSnapshot()
  })
})
