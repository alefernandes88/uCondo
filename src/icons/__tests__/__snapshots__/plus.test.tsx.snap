// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons -> Plus should render with a custom style 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="30"
  bbWidth="30"
  fill="none"
  focusable={false}
  height={30}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 30,
        "width": 30,
      },
    ]
  }
  vbHeight={20}
  vbWidth={18}
  width={30}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M10.0742 8.99219H17.8613V10.9609H10.0742V19.4336H7.94727V10.9609H0.318359V8.99219H7.94727V0.800781H10.0742V8.99219Z"
      fill={
        {
          "payload": 4294901760,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`Icons -> Plus should render with a normal style 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="20"
  bbWidth="18"
  fill="none"
  focusable={false}
  height={20}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 20,
        "width": 18,
      },
    ]
  }
  vbHeight={20}
  vbWidth={18}
  width={18}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M10.0742 8.99219H17.8613V10.9609H10.0742V19.4336H7.94727V10.9609H0.318359V8.99219H7.94727V0.800781H10.0742V8.99219Z"
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
